### ISSUE TITLE
Config Error showing in GitBash

### ISSUE THREAD

**Issue by yoshimario** [JanDeDobbeleer/oh-my-posh#3441]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What happened?

I can not get rid of the config error in the GitBash program. Powershell works fine and the windows terminal. I attached a debug screenshot. I have no idea how to fix this bug or issue.

![Screenshot_20230203_013557](https://user-images.githubusercontent.com/8993080/216475594-de150d0c-43c4-4704-9763-83e0611c7050.png)
![Screenshot_20230203_013759](https://user-images.githubusercontent.com/8993080/216475600-6b08eb28-0468-4934-8673-32a05fa21174.png)

### Theme

blue-owl

### What OS are you seeing the problem on?

Windows

### Which shell are you using?

bash

### Log output

```Shell
 paint   Git  CONFIG ERROR  oh-my-posh debug

Version: 14.0.1

Segments:

ConsoleTitle(true) -   0 ms - bash in Git
session(true)      -   0 ms -  paint 
path(true)         -   2 ms -   Git
git(false)         -   0 ms -
root(false)        -   0 ms -
exit(true)         -   0 ms -  CONFIG ERROR 
node(false)        -   0 ms -
go(false)          -   0 ms -
python(false)      -   0 ms -
shell(true)        -   0 ms - in bash
time(true)         -   0 ms - at 01:37:07

Run duration: 13.3216ms

Cache path: C:\Users\paint\AppData\Local\oh-my-posh

Config path: C:/Users/paint/blue-owl.omp.json

Logs:

[DEBUG] 01:37:07.524 shell.go:Getenv:342 → C:\Users\paint\AppData\Local
[TRACE] 01:37:07.524 shell.go:Getenv(LOCALAPPDATA) - 85.5µs
[TRACE] 01:37:07.524 shell_windows.go:CachePath() - 85.5µs
[DEBUG] 01:37:07.525 shell.go:Getenv:342 → C:/Users/paint/blue-owl.omp.json
[TRACE] 01:37:07.525 shell.go:Getenv(POSH_THEME) - 0s
[DEBUG] 01:37:07.532 shell.go:Shell:587 → process name: bash.exe
[TRACE] 01:37:07.532 shell.go:Shell() - 7.7204ms
[TRACE] 01:37:07.532 shell.go:resolveConfigPath() - 7.7204ms
[TRACE] 01:37:07.532 shell.go:Init() - 8.4755ms
[TRACE] 01:37:07.532 shell.go:Flags() - 0s
[TRACE] 01:37:07.532 config.go:loadConfig() - 0s
[TRACE] 01:37:07.532 shell.go:Flags() - 0s
[DEBUG] 01:37:07.532 shell.go:Getenv:342 → NO DATA
[TRACE] 01:37:07.532 shell.go:Getenv(OMP_CACHE_DISABLED) - 0s
[TRACE] 01:37:07.532 shell_windows.go:WindowsRegistryKeyValue(HKEY_CURRENT_USER\Software\Microsoft\Windows\DWM\ColorizationColor) - 0s
[TRACE] 01:37:07.532 shell.go:GOOS() - 0s
[TRACE] 01:37:07.532 shell.go:GOOS() - 0s
[TRACE] 01:37:07.532 shell.go:GOOS() - 0s
[TRACE] 01:37:07.532 shell.go:GOOS() - 0s
[TRACE] 01:37:07.532 shell.go:GOOS() - 0s
[TRACE] 01:37:07.532 shell.go:GOOS() - 0s
[TRACE] 01:37:07.532 shell.go:GOOS() - 0s
[TRACE] 01:37:07.532 shell.go:GOOS() - 0s
[TRACE] 01:37:07.532 shell.go:GOOS() - 0s
[TRACE] 01:37:07.532 shell.go:GOOS() - 0s
[TRACE] 01:37:07.532 shell.go:GOOS() - 0s
[TRACE] 01:37:07.532 shell.go:GOOS() - 0s
[TRACE] 01:37:07.532 shell.go:GOOS() - 0s
[TRACE] 01:37:07.532 shell.go:GOOS() - 0s
[TRACE] 01:37:07.532 shell.go:GOOS() - 0s
[TRACE] 01:37:07.532 shell.go:GOOS() - 0s
[TRACE] 01:37:07.532 shell.go:GOOS() - 0s
[TRACE] 01:37:07.532 shell.go:GOOS() - 0s
[TRACE] 01:37:07.532 shell.go:GOOS() - 0s
[DEBUG] 01:37:07.532 shell_windows.go:WindowsRegistryKeyValue:206 → ColorizationColor(DWORD): 0xC462779A
[TRACE] 01:37:07.532 shell_windows.go:Root() - 0s
[TRACE] 01:37:07.532 shell.go:Shell() - 0s
[TRACE] 01:37:07.532 shell.go:ErrorCode() - 0s
[TRACE] 01:37:07.532 shell_windows.go:IsWsl() - 0s
[TRACE] 01:37:07.533 shell.go:GOOS() - 0s
[DEBUG] 01:37:07.533 shell.go:Pwd:370 → NO DATA
[TRACE] 01:37:07.533 shell.go:Pwd() - 0s
[DEBUG] 01:37:07.533 shell_windows.go:56 → C:\Users\paint
[TRACE] 01:37:07.533 shell.go:GOOS() - 0s
[TRACE] 01:37:07.533 shell.go:GOOS() - 0s
[TRACE] 01:37:07.533 shell.go:GOOS() - 0s
[TRACE] 01:37:07.533 shell.go:GOOS() - 0s
[TRACE] 01:37:07.533 shell.go:GOOS() - 0s
[TRACE] 01:37:07.533 shell.go:GOOS() - 0s
[DEBUG] 01:37:07.533 shell.go:User:491 → paint
[TRACE] 01:37:07.533 shell.go:User() - 0s
[DEBUG] 01:37:07.533 shell.go:Host:503 → SEANLAPTOP
[TRACE] 01:37:07.533 shell.go:Host() - 0s
[TRACE] 01:37:07.533 shell.go:GOOS() - 0s
[TRACE] 01:37:07.533 shell.go:TemplateCache() - 533.5µs
[TRACE] 01:37:07.533 shell.go:Flags() - 0s
[DEBUG] 01:37:07.533 shell.go:Pwd:350 → C:\Program Files\Git
[TRACE] 01:37:07.533 shell.go:Pwd() - 0s
[DEBUG] 01:37:07.533 shell_windows.go:56 → C:\Users\paint
[TRACE] 01:37:07.533 shell.go:GOOS() - 0s
[DEBUG] 01:37:07.533 shell.go:Getenv:342 → NO DATA
[TRACE] 01:37:07.533 shell.go:Getenv(SSH_CONNECTION) - 0s
[DEBUG] 01:37:07.533 shell.go:Getenv:342 → NO DATA
[TRACE] 01:37:07.533 shell.go:Getenv(SSH_CLIENT) - 0s
[TRACE] 01:37:07.533 shell.go:TemplateCache() - 0s
[TRACE] 01:37:07.533 shell.go:TemplateCache() - 0s
[TRACE] 01:37:07.533 shell.go:Shell() - 0s
[DEBUG] 01:37:07.533 shell.go:Pwd:350 → C:\Program Files\Git
[TRACE] 01:37:07.533 shell.go:Pwd() - 0s
[DEBUG] 01:37:07.533 shell_windows.go:56 → C:\Users\paint
[TRACE] 01:37:07.533 shell.go:GOOS() - 0s
[DEBUG] 01:37:07.533 shell.go:Pwd:350 → C:\Program Files\Git
[TRACE] 01:37:07.533 shell.go:Pwd() - 0s
[TRACE] 01:37:07.533 shell.go:Shell() - 0s
[TRACE] 01:37:07.533 shell.go:Shell() - 0s
[DEBUG] 01:37:07.533 shell_windows.go:56 → C:\Users\paint
[TRACE] 01:37:07.533 shell.go:GOOS() - 0s
[TRACE] 01:37:07.533 shell.go:GOOS() - 0s
[TRACE] 01:37:07.533 shell.go:PathSeparator() - 0s
[TRACE] 01:37:07.533 shell.go:GOOS() - 0s
[TRACE] 01:37:07.533 shell.go:GOOS() - 0s
[TRACE] 01:37:07.533 shell.go:GOOS() - 0s
[TRACE] 01:37:07.533 shell.go:GOOS() - 0s
[TRACE] 01:37:07.533 shell.go:GOOS() - 0s
[TRACE] 01:37:07.533 shell.go:GOOS() - 0s
[TRACE] 01:37:07.533 shell.go:GOOS() - 0s
[TRACE] 01:37:07.533 shell.go:PathSeparator() - 0s
[TRACE] 01:37:07.533 shell.go:PathSeparator() - 0s
[TRACE] 01:37:07.533 shell.go:GOOS() - 0s
[TRACE] 01:37:07.533 shell.go:GOOS() - 0s
[TRACE] 01:37:07.533 shell.go:GOOS() - 0s
[TRACE] 01:37:07.533 shell.go:PathSeparator() - 0s
[TRACE] 01:37:07.533 shell.go:GOOS() - 0s
[TRACE] 01:37:07.533 shell.go:GOOS() - 0s
[TRACE] 01:37:07.533 shell.go:GOOS() - 0s
[TRACE] 01:37:07.533 shell.go:PathSeparator() - 0s
[TRACE] 01:37:07.533 shell.go:GOOS() - 0s
[TRACE] 01:37:07.533 shell.go:GOOS() - 0s
[TRACE] 01:37:07.533 shell.go:GOOS() - 0s
[TRACE] 01:37:07.533 shell.go:PathSeparator() - 0s
[TRACE] 01:37:07.533 shell.go:GOOS() - 0s
[TRACE] 01:37:07.533 shell.go:GOOS() - 0s
[TRACE] 01:37:07.533 shell.go:GOOS() - 0s
[TRACE] 01:37:07.533 shell.go:GOOS() - 0s
[TRACE] 01:37:07.533 shell.go:GOOS() - 0s
[TRACE] 01:37:07.533 shell.go:PathSeparator() - 0s
[TRACE] 01:37:07.533 shell.go:PathSeparator() - 0s
[DEBUG] 01:37:07.533 shell.go:Pwd:350 → C:\Program Files\Git
[TRACE] 01:37:07.533 shell.go:Pwd() - 0s
[TRACE] 01:37:07.533 shell_windows.go:IsWsl() - 0s
[TRACE] 01:37:07.533 shell.go:StackCount() - 0s
[DEBUG] 01:37:07.534 win32_windows.go:isWriteable:299 → not current user or in group
[DEBUG] 01:37:07.534 win32_windows.go:isWriteable:299 → not current user or in group
[DEBUG] 01:37:07.534 win32_windows.go:isWriteable:299 → not current user or in group
[DEBUG] 01:37:07.534 win32_windows.go:isWriteable:299 → not current user or in group
[DEBUG] 01:37:07.534 win32_windows.go:isWriteable:303 → current user is member of S-1-5-32-544
[DEBUG] 01:37:07.534 win32_windows.go:isWriteable:311 ↓
    WRITE_DAC
    WRITE_OWNER
    SYNCHRONIZE
    DELETE
    READ_CONTROL
[DEBUG] 01:37:07.534 win32_windows.go:isWriteable:313 → user has write access
[TRACE] 01:37:07.534 shell_windows.go:DirIsWritable() - 582.2µs
[TRACE] 01:37:07.534 shell.go:TemplateCache() - 0s
[TRACE] 01:37:07.535 shell.go:TemplateCache() - 0s
[TRACE] 01:37:07.535 shell.go:Shell() - 0s
[DEBUG] 01:37:07.535 shell.go:Pwd:350 → C:\Program Files\Git
[TRACE] 01:37:07.535 shell.go:Pwd() - 0s
[DEBUG] 01:37:07.535 shell_windows.go:56 → C:\Users\paint
[TRACE] 01:37:07.535 shell.go:GOOS() - 0s
[TRACE] 01:37:07.535 shell.go:GOOS() - 0s
[DEBUG] 01:37:07.536 shell.go:CommandPath:542 → C:\Program Files\Git\mingw64\bin\git.exe
[TRACE] 01:37:07.536 shell.go:CommandPath(git.exe) - 524.8µs
[TRACE] 01:37:07.536 shell.go:HasCommand(git.exe) - 524.8µs
[DEBUG] 01:37:07.536 shell.go:Pwd:350 → C:\Program Files\Git
[TRACE] 01:37:07.536 shell.go:Pwd() - 0s
[ERROR] 01:37:07.536 shell.go:HasParentFilePath:675 → CreateFile C:\.git: The system cannot find the file specified.
[TRACE] 01:37:07.536 shell.go:HasParentFilePath(.git) - 0s
[DEBUG] 01:37:07.536 shell.go:Pwd:350 → C:\Program Files\Git
[TRACE] 01:37:07.536 shell.go:Pwd() - 0s
[DEBUG] 01:37:07.536 shell_windows.go:56 → C:\Users\paint
[TRACE] 01:37:07.536 shell.go:GOOS() - 0s
[TRACE] 01:37:07.536 shell_windows.go:Root() - 0s
[DEBUG] 01:37:07.536 shell.go:Pwd:350 → C:\Program Files\Git
[TRACE] 01:37:07.536 shell.go:Pwd() - 0s
[DEBUG] 01:37:07.536 shell_windows.go:56 → C:\Users\paint
[TRACE] 01:37:07.536 shell.go:GOOS() - 0s
[TRACE] 01:37:07.536 shell.go:ErrorCode() - 0s
[TRACE] 01:37:07.536 shell.go:TemplateCache() - 0s
[TRACE] 01:37:07.536 shell.go:Shell() - 0s
[TRACE] 01:37:07.536 shell.go:Flags() - 0s
[DEBUG] 01:37:07.536 shell.go:Pwd:350 → C:\Program Files\Git
[TRACE] 01:37:07.536 shell.go:Pwd() - 0s
[DEBUG] 01:37:07.536 shell_windows.go:56 → C:\Users\paint
[TRACE] 01:37:07.536 shell.go:GOOS() - 0s
[DEBUG] 01:37:07.536 shell.go:Pwd:350 → C:\Program Files\Git
[TRACE] 01:37:07.536 shell.go:Pwd() - 0s
[DEBUG] 01:37:07.536 shell_windows.go:56 → C:\Users\paint
[DEBUG] 01:37:07.536 shell.go:Pwd:350 → C:\Program Files\Git
[TRACE] 01:37:07.536 shell.go:Pwd() - 0s
[TRACE] 01:37:07.536 shell.go:HasFiles(*.js) - 0s
[DEBUG] 01:37:07.536 shell.go:Pwd:350 → C:\Program Files\Git
[TRACE] 01:37:07.536 shell.go:Pwd() - 0s
[TRACE] 01:37:07.537 shell.go:HasFiles(*.ts) - 554.1µs
[DEBUG] 01:37:07.537 shell.go:Pwd:350 → C:\Program Files\Git
[TRACE] 01:37:07.537 shell.go:Pwd() - 0s
[TRACE] 01:37:07.537 shell.go:HasFiles(package.json) - 0s
[DEBUG] 01:37:07.537 shell.go:Pwd:350 → C:\Program Files\Git
[TRACE] 01:37:07.537 shell.go:Pwd() - 0s
[TRACE] 01:37:07.537 shell.go:HasFiles(.nvmrc) - 0s
[DEBUG] 01:37:07.537 shell.go:Pwd:350 → C:\Program Files\Git
[TRACE] 01:37:07.537 shell.go:Pwd() - 0s
[TRACE] 01:37:07.537 shell.go:HasFiles(pnpm-workspace.yaml) - 0s
[DEBUG] 01:37:07.537 shell.go:Pwd:350 → C:\Program Files\Git
[TRACE] 01:37:07.537 shell.go:Pwd() - 0s
[TRACE] 01:37:07.537 shell.go:HasFiles(.pnpmfile.cjs) - 0s
[DEBUG] 01:37:07.537 shell.go:Pwd:350 → C:\Program Files\Git
[TRACE] 01:37:07.537 shell.go:Pwd() - 0s
[TRACE] 01:37:07.537 shell.go:HasFiles(.npmrc) - 0s
[DEBUG] 01:37:07.537 shell.go:Pwd:350 → C:\Program Files\Git
[TRACE] 01:37:07.537 shell.go:Pwd() - 0s
[TRACE] 01:37:07.537 shell.go:HasFiles(.vue) - 0s
[DEBUG] 01:37:07.537 shell.go:Pwd:350 → C:\Program Files\Git
[TRACE] 01:37:07.537 shell.go:Pwd() - 0s
[DEBUG] 01:37:07.537 shell_windows.go:56 → C:\Users\paint
[TRACE] 01:37:07.537 shell.go:GOOS() - 0s
[DEBUG] 01:37:07.537 shell.go:Pwd:350 → C:\Program Files\Git
[TRACE] 01:37:07.537 shell.go:Pwd() - 0s
[DEBUG] 01:37:07.537 shell_windows.go:56 → C:\Users\paint
[DEBUG] 01:37:07.537 shell.go:Pwd:350 → C:\Program Files\Git
[TRACE] 01:37:07.537 shell.go:Pwd() - 0s
[TRACE] 01:37:07.537 shell.go:HasFiles(*.go) - 0s
[DEBUG] 01:37:07.537 shell.go:Pwd:350 → C:\Program Files\Git
[TRACE] 01:37:07.537 shell.go:Pwd() - 0s
[TRACE] 01:37:07.537 shell.go:HasFiles(go.mod) - 0s
[DEBUG] 01:37:07.537 shell.go:Pwd:350 → C:\Program Files\Git
[TRACE] 01:37:07.537 shell.go:Pwd() - 0s
[DEBUG] 01:37:07.537 shell_windows.go:56 → C:\Users\paint
[TRACE] 01:37:07.537 shell.go:GOOS() - 0s
[DEBUG] 01:37:07.537 shell.go:Pwd:350 → C:\Program Files\Git
[TRACE] 01:37:07.537 shell.go:Pwd() - 0s
[DEBUG] 01:37:07.537 shell_windows.go:56 → C:\Users\paint
[DEBUG] 01:37:07.537 shell.go:Pwd:350 → C:\Program Files\Git
[TRACE] 01:37:07.537 shell.go:Pwd() - 0s
[TRACE] 01:37:07.537 shell.go:HasFiles(*.py) - 0s
[DEBUG] 01:37:07.537 shell.go:Pwd:350 → C:\Program Files\Git
[TRACE] 01:37:07.537 shell.go:Pwd() - 0s
[TRACE] 01:37:07.537 shell.go:HasFiles(*.ipynb) - 0s
[DEBUG] 01:37:07.537 shell.go:Pwd:350 → C:\Program Files\Git
[TRACE] 01:37:07.537 shell.go:Pwd() - 0s
[TRACE] 01:37:07.537 shell.go:HasFiles(pyproject.toml) - 0s
[DEBUG] 01:37:07.537 shell.go:Pwd:350 → C:\Program Files\Git
[TRACE] 01:37:07.537 shell.go:Pwd() - 0s
[TRACE] 01:37:07.537 shell.go:HasFiles(venv.bak) - 0s
[DEBUG] 01:37:07.537 shell.go:HasFolder:430 → false
[TRACE] 01:37:07.537 shell.go:HasFolder(.venv) - 0s
[DEBUG] 01:37:07.537 shell.go:HasFolder:430 → false
[TRACE] 01:37:07.537 shell.go:HasFolder(venv) - 0s
[DEBUG] 01:37:07.537 shell.go:HasFolder:430 → false
[TRACE] 01:37:07.537 shell.go:HasFolder(virtualenv) - 0s
[DEBUG] 01:37:07.537 shell.go:HasFolder:430 → false
[TRACE] 01:37:07.537 shell.go:HasFolder(env) - 0s
[DEBUG] 01:37:07.537 shell.go:HasFolder:430 → false
[TRACE] 01:37:07.537 shell.go:HasFolder(venv-win) - 0s
[DEBUG] 01:37:07.537 shell.go:HasFolder:430 → false
[TRACE] 01:37:07.537 shell.go:HasFolder(pyenv-win) - 0s
[DEBUG] 01:37:07.537 shell.go:Pwd:350 → C:\Program Files\Git
[TRACE] 01:37:07.537 shell.go:Pwd() - 0s
[DEBUG] 01:37:07.537 shell_windows.go:56 → C:\Users\paint
[TRACE] 01:37:07.537 shell.go:GOOS() - 0s
[TRACE] 01:37:07.537 shell.go:Shell() - 0s
[TRACE] 01:37:07.537 shell.go:Flags() - 0s
[TRACE] 01:37:07.537 shell.go:TemplateCache() - 0s
[TRACE] 01:37:07.537 shell.go:TemplateCache() - 0s
[TRACE] 01:37:07.537 shell.go:Shell() - 0s
[DEBUG] 01:37:07.537 shell.go:Pwd:350 → C:\Program Files\Git
[TRACE] 01:37:07.537 shell.go:Pwd() - 0s
[DEBUG] 01:37:07.537 shell_windows.go:56 → C:\Users\paint
[TRACE] 01:37:07.537 shell.go:GOOS() - 0s
[TRACE] 01:37:07.537 shell.go:TemplateCache() - 0s
[TRACE] 01:37:07.537 shell.go:TemplateCache() - 0s
[TRACE] 01:37:07.537 shell.go:Shell() - 0s
[DEBUG] 01:37:07.537 shell.go:Getenv:342 → C:\Users\paint\AppData\Local
[TRACE] 01:37:07.537 shell.go:Getenv(LOCALAPPDATA) - 0s
[TRACE] 01:37:07.537 shell_windows.go:CachePath() - 0s
[TRACE] 01:37:07.537 shell.go:Flags() - 0s
```

**Comment by JanDeDobbeleer** [comment]
@yoshimario what does your init line look like in `.bashrc`?

**Comment by yoshimario** [comment]
its the only thing in my .bash_profile
Sincerely,
*Sean P. Kipinä*

**Comment by JanDeDobbeleer** [comment]
@yoshimario yes, but can you share the line you added there?

**Comment by yoshimario** [comment]
eval "$ (oh-my-posh init bash --config ~/blue-owl.omp.json)"
That's the line in screenshor

Sincerely,
Sean

**Comment by JanDeDobbeleer** [comment]
@yoshimario you can't use `~` in Git Bash as indicated in the docs. You need the full path as Git Bash' home reference and Windows' isn't the same.

**Comment by yoshimario** [comment]
What document I have not seen on website

**Comment by TheJordach** [comment]
> What document I have not seen on website

Greatings @yoshimario 
Please refer to the doc : https://ohmyposh.dev/docs/segments/git

- **Use the full path** 
`--config 'https://raw.githubusercontent.com/JanDeDobbeleer/oh-my-posh/main/themes/jandedobbeleer.omp.json'`
- Change your theme name accordingly

**Comment by VargasDev** [comment]
eval "$(oh-my-posh init bash --config 'https://raw.githubusercontent.com/JanDeDobbeleer/oh-my-posh/main/themes/aliens.omp.json')"

**Comment by dzalf** [comment]
> eval "$(oh-my-posh init bash --config 'https://raw.githubusercontent.com/JanDeDobbeleer/oh-my-posh/main/themes/aliens.omp.json')"

I just ran into a similar issue but in a RaspberryPi B+ where none of the local themes could properly load.

This line format was the only one that removed the CONFIG ERROR message.

Perhaps I have access problems to the themes folder?

**Comment by JanDeDobbeleer** [comment]
@dzalf oh-my-posh debug will show you what's up. But, yes on Linux your user needs to be able to access the files.

**Comment by gafda** [comment]
I have it working but it took a bit of tinkering.
I'm using Ubuntu 22.04.2 and ran the command:

`curl -s https://ohmyposh.dev/install.sh | sudo bash -s`

When I executed directly or added the following command to my .bashrc, the default theme with "CONFIG ERROR" appeared:

`eval "$(oh-my-posh init bash --config /root/.cache/oh-my-posh/themes/powerlevel10k_rainbow.omp.json)"`

The reason is that in needing sudo to run the installer, it also only allows root to access the themes.
What I've ended up doing is copying all the themes in that folder to my $HOME/.poshthemes that I've created and changed ownership with these commands:

```
sudo cp /root/.cache/oh-my-posh/themes/* ~/workspace/.poshthemes/
sudo chown -R $USER ~/workspace/.poshthemes/
```

...or if you have rsync...

`sudo rsync -a --chown=$USER /root/.cache/oh-my-posh/themes/ ~/workspace/.poshthemes/`

Now I only have to update the .bashrc to make it work again:

`eval "$(oh-my-posh init bash --config $HOME/.poshthemes/powerlevel10k_rainbow.omp.json)"`

Hope it helps everyone!
(_Tested with oh-my-posh v18.23.3_)
